pkg/dom0-ztools/rootfs/etc/init.d/010-eve-cgroup

```diff
@@ -66,7 +71,11 @@ if [ -z "${ctrd_cgroup_cpus_limit}" ] || [ "${ctrd_cgroup_cpus_limit}" = "0" ];
 fi
 
 CGROUPS="cpuset cpu cpuacct blkio memory devices freezer net_cls perf_event net_prio hugetlb pids systemd "
-EVESRVICES="sshd edgeview wwan wlan lisp guacd pillar vtpm watchdog xen-tools newlogd "
```

Comment:
If we already are fixing this, can we please correct the type for `EVESRVICES` to `EVESERVICES`?

Reply:
I can fix that.